### ISSUE TITLE
MIMIC on Azure

### ISSUE THREAD

**Issue by Mkang1204** [MIT-LCP/mimic-code#826]
### Prerequisites

* [X ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.physionet.org/about/mimic/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Dear Developers, 

Thanks for setting up the database on GCP and AWS! I wonder if you have a plan to set on Azure as well? I am sure many researchers use Azure too.

Best,
Marjorie

**Comment by tompollard** [comment]
Hi @Mkang1204, we would like to integrate PhysioNet/MIMIC with Azure too and we've had some brief discussion about doing this, but there are no immediate plans. We'll post here if/when this situation changes!

**Comment by alistairewj** [comment]
Yep and to add on to that if anyone wants to contribute code to build MIMIC on the most appropriate Azure service we'd welcome a PR :)

**Comment by Mkang1204** [comment]
Thank you @tompollard and @alistairewj for your quick reply! Glad to hear that Azure has been discussed before and I am happy to help as I can when there is a plan.
